docs/guide/features.md

```diff
@@ -565,12 +565,16 @@ const moduleUrls = import.meta.glob('./dir/*.svg', {
 ```ts
 // code produced by vite:
 const moduleStrings = {
-  './dir/foo.svg': () => import('./dir/foo.js?raw').then((m) => m['default']),
-  './dir/bar.svg': () => import('./dir/bar.js?raw').then((m) => m['default']),
+  './dir/bar.svg': () =>
+    import('./dir/bar.svg?import&raw').then((m) => m['default']),
```

Comment:
```suggestion
    import('./dir/bar.svg?raw').then((m) => m['default']),
```
I think we can keep hiding the fact of `?import` here. The order may be relied by the users but users shouldn't need to care about `?import` and showing this may lead to confusion.

Reply:
Makes sense, I hadn't noticed this before.